orchestrator-salt/src/main/resources/salt/salt/metering/init.sls

```diff
@@ -14,7 +14,7 @@
 install_metering_rpm_manually:
   cmd.run:
     - name: "rpm -i {{ metering_rpm_location }}"
-    - onlyif: "! rpm -q {{ metering_package_name }}"
+    - onlyif: "! test -e /opt/metering-heartbeat"
```

Comment:
This is more error prone if the directory ever get's moved.
Can't we rather check for systemd unit file or systemd service name?

Reply:
I think we should remove the whole installation